utils/typography.js

```diff
@@ -28,7 +28,7 @@ const typography = new Typography({
     'Verdana',
     'sans-serif'
   ],
-  scale: 1.5,
+  scaleRatio: 1.5,
```

Comment:
Ah that looks much better.  Only issue now is the spacing under the logo, which definitely is within my capabilities to figure out.  I'm done for the night right now, but will get that in and merged over the weekend.  Thanks so much!